tests/unit/config/hse_gparams_test.c

```diff
@@ -155,7 +159,7 @@ MTF_DEFINE_UTEST_PRE(hse_gparams_test, perfc_level, test_pre)
     ASSERT_EQ(PARAM_TYPE_U8, ps->ps_type);
     ASSERT_EQ((uintptr_t)ps->ps_convert, (uintptr_t)param_default_converter);
     ASSERT_EQ((uintptr_t)ps->ps_validate, (uintptr_t)param_default_validator);
-    ASSERT_EQ(PERFC_LEVEL_DEFAULT, params.gp_perfc_level);
+    ASSERT_EQ((uintptr_t)ps->ps_stringify, (uintptr_t)param_default_stringify);
```

Comment:
Curious...

Reply:
Looks like that was just lost in a rebase. Will fix